File: Common/InMotionGIT.Common/Services/Contracts/Source.cs
using System;
using System.Runtime.Serialization;

namespace InMotionGIT.Common.Services.Contracts
{

    [DataContract()]
    public class Source
    {

        [DataMember]
        public string Ip { get; set; }

        [DataMember]
        public DateTime EffectDate { get; set; }

        [DataMember]
        public int Count { get; set; }

    }

}

File: Common/InMotionGIT.Common - Copy/Services/Contracts/Source.vb
Imports System.Runtime.Serialization

Namespace Services.Contracts

    <DataContract()>
    Public Class Source

        <DataMember>
        Public Property Ip() As String

        <DataMember>
        Public Property EffectDate() As Date

        <DataMember>
        Public Property Count() As Integer

    End Class

End Namespace